thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/v2/ResourceUtils.java

```diff
@@ -283,5 +287,41 @@ private static void putExternalLinkTimeContext(long startTime, long endTime, Str
     }
   }
 
+  /**
+   * Resolves the (convoluted) anomaly feedback properties into a clear classification of
+   * {@code NONE, TRUE_POSITIVE, FALSE_POSITIVE, TRUE_NEGATIVE, FALSE_NEGATIVE}.
+   *
+   * @param anomaly anomaly dto
+   * @return feedback classification
+   */
+  public static AnomalyClassificationType getStatusClassification(MergedAnomalyResultDTO anomaly) {
+    if (anomaly.getAnomalyResultSource() != null) {
```

Comment:
As part of moving the anomaly report button to the dashboard. We will need a new resolution type of `User Reported + True anomaly`. Not sure it will need change here.

Reply:
Yes, that's a false negative. (TE prediction negative, User condition positive)